pkg/scheduling/requirements.go

```diff
@@ -220,15 +220,25 @@ func editDistance(s, t string) int {
 }
 
 func labelHint(r Requirements, key string, allowedUndefined sets.Set[string]) string {
+	splitKey := strings.Split(key, "/")
+	suffix := splitKey[1]
 	for wellKnown := range allowedUndefined {
 		if strings.Contains(wellKnown, key) || editDistance(key, wellKnown) < len(wellKnown)/5 {
 			return fmt.Sprintf(" (typo of %q?)", wellKnown)
 		}
+		if strings.Contains(wellKnown, suffix) {
+			prefix := strings.Split(wellKnown, "/")[0]
+			return fmt.Sprintf(" (typo of %q?)", prefix)
+		}
 	}
 	for existing := range r {
 		if strings.Contains(existing, key) || editDistance(key, existing) < len(existing)/5 {
 			return fmt.Sprintf(" (typo of %q?)", existing)
 		}
+		if strings.Contains(existing, suffix) {
+			prefix := strings.Split(existing, "/")[0]
+			return fmt.Sprintf(" (typo of %q?)", prefix)
```

Comment:
```suggestion
			return fmt.Sprintf(" (typo of %q?)", wellKnown)
```

We should give the entire key, not just the prefix. This also means that you can remove L239